pw_mgr/lib/crypto_utils.py

```diff
@@ -1,19 +1,26 @@
+import os
 from cryptography.fernet import Fernet
 
+"""load key.key and encrypt password data"""
+def load_key():
+    with open("key.key", "rb") as key_file:
+        print("Encryption key detected. Starting program...")
+        return Fernet(key_file.read())
+
+def create_key():
+    with open("key.key", "wb") as key_file:
+        key_file.write(Fernet.generate_key())
+        print("Encryption Key generated successfully!")
+
 
 """check if key.key exists. if not, create key.key. if yes, open key.key."""
 def init_key():
-    try:    
-        f = open("key.key")
-        print("Encryption key detected. Starting program...")
-        print("\n")
-        f.close()
-    except IOError:
-        with open ("key.key", "wb") as key_file:
-            key = Fernet.generate_key()
-            key_file.write(key)
-        print("Encryption Key generated successfully!")
+    if not os.path.exists("./key.key"):
+        create_key()
 
+    return load_key()
```

Comment:
This is really helpful to see how you approached this. Breaking down everything into granular functions so they can be nested and called within higher functions. I see what you meant on our last call about limiting the number of lines in a function and limiting the number of functions in a script, etc.